src/main/java/org/jabref/gui/fieldeditors/JournalEditorViewModel.java

```diff
@@ -22,8 +22,11 @@ public void toggleAbbreviation() {
             return;
         }
 
-        if (journalAbbreviationRepository.isKnownName(text.get())) {
-            Optional<String> nextAbbreviation = journalAbbreviationRepository.getNextAbbreviation(text.get());
+        // Ignore brackets when matching abbreviations.
+        final String name = StringUtil.ignoreCurlyBracket(text.get());
+
+        if (journalAbbreviationRepository.isKnownName(name)) {
```

Comment:
I noticed that some abbreviations in JabRef's built-in list do contain curly brackets (for formating). E.g.:
`Foundations and Trends ${}^\circledR$ in Signal Processing`
If one of those were used, it would not match anymore because the brackets were removed from the name that we are looking for but they are still present in the abbreviation. I think you should remove it from the abbreviation name as well.
https://github.com/JabRef/jabref/blob/475b2989ffa8ec61c3327c62ed8f694149f83220/src/main/java/org/jabref/logic/journals/JournalAbbreviationRepository.java#L31-L36

Reply:
It seems as though the built-in list is checked via a containsKey() call looking for the full name (JournalAbbreivationRepository.java line 59), meaning that the brackets would have to be removed from the key/abbreviation name itself in the default list as opposed to an ignoreCurlyBracket() call. Is this correct?

Reply:
Ah, the built-in entries are stored in a hash map? Then there is nothing we can do. One could try twice, once with the brackets removed and once with the original, but that would not be a good solution.